sysutils/osxbom/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+# My repo is ahead of the main one in terms of versions:
+github.setup        cooljeanius osxbom 0.0.5
+github.tarball_from archive
+categories          sysutils
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           darwin
```

Comment:
`platforms darwin` is implicit in MacPorts now, so this line can be removed.

Reply:
ok, removed in 6c41a84.